components/UI/Card/components/Label/Label.module.css

```diff
@@ -16,6 +16,12 @@
 
 .label__wrapper_clickable {
     cursor: pointer;
+    transition: opacity 150ms ease-out;
+    will-change: opacity;
```

Comment:
Are you sure you need this optimization?

Isn't the animated `opacity` running on the GPU fast enough?